randovania/games/am2r/pickup_database/item_data.json

```diff
@@ -26,8 +26,8 @@
 	},
 	"Locked Power Bomb Expansion": {
 		"sprite_speed": 0.2,
-		"text_desc": "Power Bomb capacity {Locked Power BombsChanged} by {Locked Power Bombs}",
-		"text_header": "Got Power Bomb Tank"
+		"text_desc": "Power Bomb capacity {Locked Power BombsChanged} by {Locked Power Bombs} - Launcher still required",
+		"text_header": "Got Locked PowerBomb Tank"
```

Comment:
Don't know where that will be used but it looks weird to me that it is using `PowerBomb` and `Power Bomb` in the text.
Same for super missile

Is this intentional?

Reply:
Yup. The header space is a tad limiting in-game, so this is an anti-clipping measure. also the reason why super missile is missing "tank".